testing-basics-garden-project/garden-test004.js

```diff
@@ -0,0 +1,6 @@
+import { Garden } from '../src/Garden';
+const chai = require("chai");
+const assert = chai.assert;
+
+// Define a new Garden object called myGarden using let instead of var or const
+// I'll explain why later..
```

Comment:
We could probably just shorten this to the first sentence:

```
Define a new `Garden` object called `myGarden` using `let` instead of `var` or `const`.
```